packages/redux/src/entities/types/productListing.types.ts

```diff
@@ -7,8 +7,10 @@ import type {
 } from '@farfetch/blackout-client';
 import type { ProductEntity } from './product.types.js';
 
+export type FacetGroupWithListingHash = FacetGroup & { hash?: string };
+
 export type FacetGroupsNormalized = Array<
-  Omit<FacetGroup, 'values'> & {
+  Omit<FacetGroupWithListingHash, 'values'> & {
```

Comment:
Keep FacetGroup here as I do not think we should expose the hash property to consumers yet.

Reply:
Right, I exposed it here and also in the `FacetEntity`, but it's not needed. Fixed!